src/js/engagement_view/src/components/engagementView/sidebar/tables/lensTable/lensTable.tsx

```diff
@@ -71,6 +73,8 @@ export const lensTable = (
                                     lens_type={lens.lens_type}
                                     score={lens.score}
                                     setLens={setLens}
+                                    selectedId={selectedIdState}
+                                    setSelectedLens={setSelectedIdState}
```

Comment:
There's a mismatch in these state and setter names.

IMO this should look like
```
                                    selectedLensId={selectedLensIdState}
                                    setSelectedLensId={setSelectedLensIdState}
```

a `number` named `id` tells me nothing about how that variable/state is used; I almost always try to label my IDs (lens_id, page_id, user_id, etc).

Reply:
done